src/mots/config.py

```diff
@@ -42,6 +42,11 @@
     )
 )
 
+# A list of nicks allowed to be used without a bugzilla ID.
+ALLOWED_NICK_ONLY = [
+    "TLMC",
+]
```

Comment:
Would it be possible to get this list out of the mots.yaml file itself, e.g., from an array in a separate top-level key?

This would be closer to the original suggestion in the bug, but I might not be aware of follow-up discussions.

Reply:
So we wanted the hardcoding to be harder to change, to at least add some friction whenever a new alias is needed. The "allow list" is not mentioned in the bug but is a decision that was reached after offline discussions.

However, the suggestion in the bug itself basically uses yaml anchors/references, which _can_ still be used here. The only difference is that it would have to be placed in a different list other than "people". This isn't documented (yet) but would be largely a matter of convention since there is no significance to where these anchors go at this time (as long as they are not in the "people" dictionary which syncs with bugzilla).

```yaml
...
aliases:
    - &TLMC
      nick: TLMC
...
        owners:
          - *TLMC

```